staging/src/k8s.io/api/networking/v1/generated.proto

```diff
@@ -404,7 +404,11 @@ message NetworkPolicyPort {
   optional k8s.io.apimachinery.pkg.util.intstr.IntOrString port = 2;
 }
 
-// NetworkPolicySpec provides the specification of a NetworkPolicy
+// NetworkPolicySpec provides the specification of a NetworkPolicy.  A suite of network
+// policies allow traffic ONLY if BOTH the egress AND the ingress are allowed.  For example,
+// if ingress is allowed but egress is not, or egress is allowed but ingress is not, the traffic
+// will not be allowed.  See the Ingress and Egress fields' documentation for more information
+// on when ingress and egress are allowed.
```

Comment:
I find it a bit confusing :), let me ping  @danwinship that he is pretty good at this